pkg/network/protocols/http/etw_http_service.go

```diff
@@ -1427,7 +1433,7 @@ func (hei *EtwInterface) OnStart() {
 	initializeEtwHttpServiceSubscription()
 	httpServiceSubscribed = true
 	var err error
-	iisConfig, err = winutil.NewDynamicIISConfig()
+	iisConfig, err = iisconfig.NewDynamicIISConfig()
```

Comment:
Yeah, from the previous comment iisConfig may not be fully initialized if at all and in theory you may crash on a first event.

Reply:
that is my theory of course :)

Reply:
from previous comment, iisconfig has its own locking to ensure it's consistent.

Reply:
ah, ok.  Now I get what you're saying.
And yes you're right (and worse, if the iisconfig init returns an error, we'll still dereference the nil pointer as you point out).  Fixing.